app/views/books/index.html.haml

```diff
@@ -1,4 +1,7 @@
-%h1 My library
+- if @user == current_user
+  %h1 My library
+- else
+  %h1= "#{@user.decorate.display_full_name}'s  library"
```

Comment:
I logged in on staging via my Google account, and for some reason, when I go to someone else's library, their full name is blank. Do you know why this happens?

Reply:
so the problem is that they didn't provide their names..  I think it could be handled in different ways: 
- after sign  up I should redirect user to `My profile` page with prompt 'Please fill in your name'
- render user's email instead, but in the future I wouldn't like to make users' emails public

Reply:
as we have discussed, I'll change registration form so that all users will have to provide their names.